components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -192,7 +201,7 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
               process.env.REACT_APP_VERSIONS.split(',').map((version, i) => (
                 <MenuItem key={i} value={version}>{version}</MenuItem>
               )) :
-              versionList.map((version, i) => (
+                this._versions.map((version, i) => (
```

Comment:
Currently ```process.env.REACT_APP_VERSIONS``` has higher priority to set version list.
You probably want to use new param as high priority one for version list

Reply:
when and how was it set? is it a config parameter in click-to-deploy application that's not exist in this repo?

Reply:
anyway always respect the query string as the first option. done

Reply:
REACT_APP_VERSIONS is set through configmap in k8s cluster.